includes/widgets/rating.php

```diff
@@ -217,7 +217,14 @@ protected function register_controls() {
 	}
 
 	protected function get_rating_value(): float {
-		return floatval( $this->get_settings_for_display( 'rating_value' ) );
+		$initial_value = $this->get_rating_scale();
+		$rating_value = $this->get_settings_for_display( 'rating_value' );
+
+		if ( '' === $rating_value ) {
+			$rating_value = $initial_value;
+		}
+
+		return floatval( $rating_value );
```

Comment:
Can we do a return and check in one-line with ternary operator?

Reply:
I prefer to keep it this way, to keep it more readable.
Originally  I used a ternary operator, but then Louis asked me to change it :).